docs/Plugins.md

```diff
@@ -185,11 +185,59 @@ The matching simplestream config to send audio from talkgroup 58918 to TCP port
         }
 ```
 
+## Community Plugins
+Community plugins can extend the features of Trunk Recorder and allow customized workflows or analysis.  
+> As new plugins are developed, authors are encouraged to add to the below tables by submitting a PR to this document.
```

Comment:
This creates an unfortunate dependency to PR against trunk/main. Maybe we could automatically pull in any README.md or just say "refer to user plugin documentation" in this section.

Reply:
Probably the easiest way is to just link a trunk-recorder wiki page and let people add things themselves.